projects/plugins/jetpack-mu-wpcom/jetpack-mu-wpcom.php

```diff
@@ -31,3 +31,18 @@
 	exit;
 }
 
+/**
+ * Load the Coming Soon feature.
+ */
+function load_coming_soon() {
+	if (
+		( defined( 'WPCOM_PUBLIC_COMING_SOON' ) && WPCOM_PUBLIC_COMING_SOON ) ||
+		apply_filters( 'a8c_enable_public_coming_soon', false )
+	) {
+		require_once __DIR__ . '/src/features/coming-soon/coming-soon.php';
+	}
+}
+// Todo: once coming-soon is removed from ETK, we can remove this check.
+if ( has_action( 'plugins_loaded', 'A8C\FSE\load_coming_soon' ) === false ) {
+	add_action( 'plugins_loaded', 'load_coming_soon' );
```

Comment:
Should we put this plugin's functions in a namespace too?

Reply:
6dfaeaea72e06146ffe62227d97904ed09619462